components/prism-sas.js

```diff
@@ -8,6 +8,16 @@
 		alias: 'number'
 	};
 
+	var macroVariable = {
+		pattern: /&[a-z_][a-z_1-9]*/i
```

Comment:
`0` is not allowed?

Reply:
It is allowed. Thanks for catching that.